worker/src/main/java/gov/cms/ab2d/worker/processor/PatientClaimsCollector.java

```diff
@@ -65,7 +64,7 @@ public void filterAndAddEntries(IBaseBundle bundle) {
         // Perform filtering actions
         BundleUtils.getEobResources(bundleEntries).stream()
                 // Filter by date
-                .filter(resource -> skipBillablePeriodCheck || FilterOutByDate.valid(resource, attestationDate, earliestDate, claimsRequest.getCoverageSummary().getDateRanges()))
+                .filter(resource -> Contract.hasTestDateIssues(claimsRequest.getContractNum()) || FilterOutByDate.valid(resource, attestationDate, earliestDate, claimsRequest.getCoverageSummary().getDateRanges()))
```

Comment:
Not thrilled about Contract exposing static methods.

PatientClaimsRequest could easily store a reference to a contract (instead of the number), allowing for the call to be simply claimsRequest.getContract().hasTestDateIssues().  That helps in hiding more details.  Let Contract manage the contract number.